### PR TITLE
Feat/matching module sub components

### PR DESCRIPTION
**Pull request by michalisKout**
## Description
Exposes two components from Asset matching: 
- SectionHeader 
- ActionsToolbox

Added two stories extra in storybook.

We can reuse these components for handling bulk actions in any project.

## Screenshot

In the following image, you can check that the bulk actions section and the header section of the AssetMatching are identical. Therefore we can directly use the SectionHeader component to handle bulk actions

Screenshot 2021-04-06 at 2 56 53 PM

## Test Plan

update snapshots

### CODE REVIEW

File: src/components/AssetMatching/components/ActionsToolbox/SecondaryActions.tsx [hunk 1]
```diff
@@ -8,17 +8,22 @@ import ClickAwayListener from 'components/utils/ClickAwayListener';
 
 interface Props {
   secondaryActions: MatchingAction[];
+  actionButtonFill?: boolean;
```
Comment (panvourtsis) [inline src/components/AssetMatching/components/ActionsToolbox/SecondaryActions.tsx:11]: can we use our definition of naming bool? https://orfium.atlassian.net/wiki/spaces/ENG/pages/1335591044/General+Code+Rules

File: src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx [hunk 2]
```diff
@@ -14,21 +14,33 @@ interface Props {
   score?: string | number;
   matchingActions?: MatchingAction[];
   styleType: formFieldStyles;
+  actionButtonFill?: boolean;
+  customCheckboxContent?: JSX.Element;
```
Comment (panvourtsis) [inline src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx:18]: as this is JSX i would expect capital case

**Review by panvourtsis — CHANGES_REQUESTED** [review]

File: src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx (hunk 2 above)
Comment (michalisKout) [reply src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx:18]: Should we use capital case in variables that hold jsx element? 
I thought only functional components should start with capital letter.

for example: 
```
we use button like this:
<Button iconLeft={...} />

iconLeft holds a jsx element.
```

Comment (panvourtsis) [reply src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx:18]: sorry my fault you are right only for components

**Review by panvourtsis — APPROVED** [review]

**Pull request merged**